media-proxy/src/proxy_context.cc

```diff
@@ -345,18 +369,16 @@ void ProxyContext::ParseSt20RxOps(const mcm_conn_param* request, struct st20p_rx
     strlcpy(ops_rx->port.port[MTL_PORT_P], getDevicePort().c_str(), MTL_PORT_MAX_LEN);
     ops_rx->port.num_port = 1;
     if(request->payload_type_nr == 0 ) {
-        ops_rx->port.payload_type = 112;
+        ops_rx->port.payload_type = 96;
```

Comment:
Can we use a const not a hardcoded number here?

Reply:
Fixed - added preprocessor const from MTL repository.